test/tall/invocation/chain_property.stmt

```diff
@@ -50,4 +50,22 @@ avian
     .equine
     .feline
     .piscine
-    .method();
\ No newline at end of file
+    .method();
+>>> Prefer splitting an assignment over splitting a pure property chain.
+variable = avian.bovine.canine.equine.feline;
+<<<
+variable =
+    avian.bovine.canine.equine.feline;
```

Comment:
How about a case where the RHS is too long to fit in one line? Will it move the initial selector back to the previous line?

Reply:
Oh, good question! Yes, in that case it will split the property chain and put the first selector on the same line as the `=`. Added a test.